### PR TITLE
👷‍♀️ FIX: aggregate query buttons and rename "chainPredicate" to "and"

### PR DESCRIPTION
**Pull request by kenjinp**
For Aggregate Views, those little query buttons wouldn't show up. Now they will!

changed some names because they make much more sense now

### CODE REVIEW

**Comment by kenjinp** [comment]
I also fixed the styling for query view results, and querying Aggregated Views now works too

File: src/shared/utils/nexus-maybe.ts [hunk 1]
```diff
@@ -3,8 +3,9 @@ import { Resource } from '@bbp/nexus-sdk-legacy';
 const NEXUS_FILE_TYPE = 'File';
 const VIEW = 'View';
 const ES_VIEW = 'ElasticSearchView';
+const AGGREGATE_ES_VIEW = 'AggregateElasticSearchView';
+const AGGREGATE_SPARQL_VIEW = 'AggregateSparqlView';
 const SPARQL_VIEW = 'SparqlView';
-
 const DEFAULT_ELASTIC_SEARCH_VIEW_ID = 'nxv:defaultElasticSearchIndex';
 
 const isOfType = (type: string) => (resource: Resource) =>
```
Comment (julienmachon) [inline src/shared/utils/nexus-maybe.ts]: Why is it still using `Resource` from the legacy sdk?

Comment (kenjinp) [reply src/shared/utils/nexus-maybe.ts]: it was made before the conversion to v1.1. Should it be any object, or a new ResourceResponseCommon or so?

Comment (julienmachon) [reply src/shared/utils/nexus-maybe.ts]: Yes, either you create a new type or you use the new SDK one. But if you add new functionality or update something in general, make sure you refactor and remove the legacy dependencies.

Comment (kenjinp) [reply src/shared/utils/nexus-maybe.ts]: Also the views that is using this have the resources in the format such as resource[`@type`] is actually resource.type, so without changing that up the chain for the Resource view we'll have problems using this.

**Pull request closed without merging**